manageiq-operator/api/v1alpha1/helpers/miq-components/httpd_conf.go

```diff
@@ -39,35 +39,35 @@ Options SymLinksIfOwnerMatch
   RequestHeader set Host %[1]s
   RequestHeader set X-Forwarded-Host %[1]s
 
+  # Send API requests to the API pods
+  ProxyPass /api %[4]s://web-service:3000/api
+  ProxyPassReverse /api %[4]s://web-service:3000/api
+
+  # Send Notifications requests to the UI pods
   RewriteCond %%{REQUEST_URI}     ^/ws/notifications [NC]
   RewriteCond %%{HTTP:UPGRADE}    ^websocket$ [NC]
   RewriteCond %%{HTTP:CONNECTION} ^Upgrade$   [NC]
   RewriteRule .* %[2]s://ui:3000%%{REQUEST_URI}  [P,QSA,L]
   ProxyPassReverse /ws/notifications %[2]s://ui:3000/ws/notifications
 
-  RewriteCond %%{REQUEST_URI} !^/api
+  # Send Console sessions to the remote-console pods
+  RewriteCond %%{REQUEST_URI}     ^/ws/console [NC]
+  RewriteCond %%{HTTP:UPGRADE}    ^websocket$  [NC]
+  RewriteCond %%{HTTP:CONNECTION} ^Upgrade$    [NC]
+  RewriteRule .* ws://remote-console:3000%%{REQUEST_URI}  [P,QSA,L]
+  ProxyPassReverse /ws/console ws://remote-console:3000/ws/console
 
+  # Send everything else that is not handled locally to the UI pods
+  RewriteCond %%{REQUEST_URI} !^/api
```

Comment:
We probably don't need this if we handle the API requests immediately

Reply:
Won't this prevent the next handful of lines?  I'm not sure what this line is meant to do.